test/e2e/apimachinery/BUILD

```diff
@@ -39,13 +41,15 @@ go_library(
         "//staging/src/k8s.io/api/extensions/v1beta1:go_default_library",
         "//staging/src/k8s.io/api/rbac/v1:go_default_library",
         "//staging/src/k8s.io/api/rbac/v1beta1:go_default_library",
+        "//staging/src/k8s.io/api/scheduling/v1beta1:go_default_library",
```

Comment:
is this needed?

Reply:
it's used to create PriorityClass, so yes it's needed. (otherwise bazel will remove it)